cshared/main.go

```diff
@@ -39,6 +39,8 @@ import (
 	"github.com/googleapis/enterprise-certificate-proxy/client"
 )
 
+var Version = "dev"
```

Comment:
Can you add a comment saying this will be overwritten by the ldflag in the build script?

Reply:
done